crates/ruff/src/rules/flake8_simplify/rules/ast_with.rs

```diff
@@ -77,7 +83,14 @@ pub(crate) fn multiple_with_statements(
             return;
```

Comment:
Took me way too long to figure this out:
```suggestion
            // Make sure with fix from top to bottom for nested with statements, e.g. for
            // ```python
            // with A():
            //     with B():
            //         with C():
            //             print("hello")
            // ```
            // suggest
            // ```python
            // with A(), B():
            //     with C():
            //         print("hello")
            // ```
            // but not the following
            // ```python
            // with A():
            //     with B(), C():
            //         print("hello")
            // ```
            return;
```
I'd place it outside the if let but github only allows me to place comments here